overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -123224,7 +123224,7 @@
           "repeat_relative:1": 1,
           "ismain:1": 0,
           "name:8": "Better than Dirt",
-          "desc:8": "Much better than Dirt in fact...! Craft some additional Elemental Soil and upgrade some batches of them to all 4 different types of Elemental Soil:\n\n- Drop it in Water to upgrade it to Aqueous Soil,\n- Drop it in Lava to upgrade it to Magmatic Soil,\n- Drop it above Y\u003d90 to upgrade it to Caelic Soil,\n- Drop it below Y\u003d30 to upgrade it to Terran Soil.\n\nEach type of Soil has its benefits (read their tooltip).\n\nNote: The effect of the Aqueous Soil can be disabled by placing a piece of Gravel underneath the Soil.\nNote: Aqueous Soil is particularly good for early crop farms, before you have unlocked farmers. Use a Hopper Minecart or the Roots Ritual to collect the drops."
+          "desc:8": "Much better than Dirt in fact...! Craft some additional Elemental Soil and upgrade some batches of them to all 4 different types of Elemental Soil:\n\n- Drop it in Water to upgrade it to Aqueous Soil,\n- Drop it in Lava to upgrade it to Magmatic Soil,\n- Drop it above Y\u003d90 to upgrade it to Caelic Soil,\n- Drop it below Y\u003d30 to upgrade it to Terran Soil.\n\nEach type of Soil has its benefits (read their tooltip).\n\nNote: The effect of the Aqueous Soil can be disabled by placing a piece of Gravel underneath the Soil.\nNote: Aqueous Soil is particularly good for early crop farms, before you have unlocked farmers. Use a Hopper Minecart or the Roots Ritual to collect the drops. You can also use other methods to pick up drops from the Aqueous Soil since it drops outputs into inventories below it. Some good options to make good use of this mechanic is by using Drawers, Immersive Engineering Conveyer Belts, and more."
```

Comment:
please reword `...and more` into `include...`; specify 2x1 Drawers and how that whole process works; conveyors apparently dont work properly; make it a list of options rather than "Hopper Minecraft || Ritual. Also Inventory below"